### ISSUE TITLE
lots of error

### ISSUE THREAD

**Issue by dipenpatel235** [remoteinterview/compilebox#42]
hello..
i have try to used your docker images but lots of error not success to working.
So please verify it and also possible so make it working script for Ubuntu16 or centos without docker 

Thanks

**Comment by AndyChordiya** [comment]
yup, i am facing same issue.. i have install all things properly, my temp folder is created, but I am unable to get output.

Please look into it..

Thanks
